Library/Homebrew/cask/lib/hbc/dsl/appcast.rb

```diff
@@ -9,6 +11,24 @@ def initialize(uri, parameters = {})
         @checkpoint     = @parameters[:checkpoint]
       end
 
+      def calculate_checkpoint
+        result = SystemCommand.run("/usr/bin/curl", args: ["--compressed", "--location", "--user-agent", URL::FAKE_USER_AGENT, @uri], print_stderr: false)
+
+        checkpoint = if result.success?
+          processed_appcast_text = result.stdout.gsub(%r{<pubDate>[^<]*</pubDate>}, "")
+
+          # This step is necessary to replicate running `sed` from the command line
```

Comment:
The problem is that some casks have `<pubDate>` and `</pubDate>` on different lines, so it doesn’t actually get removed. By removing all newlines, we make sure `<pubDate>[^<]*</pubDate>` always works.

I guess we can just setup the regex to work multi-line, yes.